packages/terra-menu/src/_MenuUtils.js

```diff
@@ -15,8 +15,11 @@ const isFullScreen = (isHeightBounded, isWidthBounded, boundingFrame, popupWidth
   return isHeightBounded && (width >= maxWidth || isWidthBounded);
 };
 
+const isMac = () => navigator.userAgent.indexOf('Mac') !== -1 && navigator.userAgent.indexOf('Win') === -1;
```

Comment:
is this property common across all the browsers used in mac ?

Reply:
yes this is added due to the gap in  VO response  where it fails to announce index of items